docs/references/amp-constants.md

```diff
@@ -0,0 +1,59 @@
+# AMP Constants
+
+Amp Constants are the information passed from the AMP Engine to the tool
```

Comment:
TW review:

AMP Constants are the information passed from the AMP engine to the tool.

Reply:
We incorporated the change.